plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -1451,36 +1459,16 @@ $.extend(DataTable.prototype, UIControl.prototype, {
 
     //Apply some miscelleaneous style to the DataTable
     applyCosmetics: function (domElem) {
-        var self = this;
-
-        // Add some styles on the cells even/odd
-        // label (first column of a data row) or not
-        $("th:first-child", domElem).addClass('label');
-        $("td:first-child", domElem).addClass('label');
-
-        var metadata = this.getReportMetadata();
-
-        if (self.param.flat == "1" && self.param.show_dimensions == "1" && metadata.dimensions && Object.keys(metadata.dimensions).length > 1) {
-            for (var i = 1; i < Object.keys(metadata.dimensions).length; i++) {
-                $("th:nth-child("+(i+1)+")", domElem).addClass('label');
-                $("td:nth-child("+(i+1)+")", domElem).addClass('label');
-            }
-        }
-
-        $("tr td", domElem).addClass('column');
+        // empty
     },
 
     handleColumnHighlighting: function (domElem) {
-        if (!this.canHandleRowEvents(domElem)) {
```

Comment:
With ~1250 rows visible it's slower, but not because of `handleColumnHighlighting()`. If I simply `return false;` at the start, the speed of hovering doesn't seem to change. I think you'll have to try it yourself to see... should be on demo2 when I push again.